complete/grails-app/conf/logback.groovy

```diff
@@ -34,3 +34,4 @@ if (Environment.isDevelopmentMode() && targetDir != null) {
     logger("StackTrace", ERROR, ['FULL_STACKTRACE'], false)
 }
 root(ERROR, ['STDOUT'])
+logger 'grails.mock.basics', INFO, ['STDOUT'], false
```

Comment:
I replaced a print statement with a log.info statment. You will need to register a logger at level info to see that output